src/domain/schemas/user-schemas.js

```diff
@@ -57,3 +57,37 @@ export const loginBodySchema = joi.object({
   email: joi.string().required(),
   password: joi.string().required()
 });
+
+const storageReminderDBSchema = joi.object({
+  timestamp: joi.date().required(),
+  createdBy: idOrKeySchema.required()
+});
+
+const storageDBSchema = joi.object({
+  plan: joi.string().allow(null).required(),
```

Comment:
Should we make a small separate ticket to turn this into `planId` ?

Reply:
Nice idea, I will create one! :+1: